bot/scrypt_tunes.py

```
@@ -368,9 +539,9 @@ async def chat_song_request(self, ctx, song, song_uri, album: bool, requests=Non
 
             elif re.match(self.URL_REGEX, song_uri):
                 if 'spotify' in song_uri:
-                    if '.link/' in song_uri:  # todo: better way to handle this?
+                    if '.link/' in song_uri:  
                         ctx.send(
-                            f'@{ctx.author.name} Mobile link detected, attempting to get full url.')  # todo: verify this is sending?????
+                            f'@{ctx.author.name} Mobile link detected, attempting to get full url.')  
                         req_data = req.get(
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)